recipes/tpl/all/conandata.yml

```
@@ -0,0 +1,4 @@
+sources:
+  "1.6.1.cci.20210302":
```

the new convention for the cci version format is still not in place as commented here https://github.com/conan-io/conan-center-index/pull/24411#pullrequestreview-2138128755

Basically, we would need to drop support of conan 1 first so we can have it as an official version format to be fully compatible with version ranges. Sorry if it was not clear
```suggestion
  "cci.20210302":
```